website/templates/emails/reviews_submission_status.html.mako

```diff
@@ -1,58 +1,75 @@
-                Admins may edit the ${reviewable.provider.preprint_word} and contact
-                the moderator at ${provider_contact_email} to resubmit.
-            % else:
-                been accepted by the moderator and ${'remains' if was_pending else 'is now'} discoverable to others.
-            % endif
-        % endif
 
-        % if notify_comment:
-            The moderator has also provided a comment that is only visible to contributors
-            of the ${reviewable.provider.preprint_word}, and not to others.
+            % if notify_comment:
+                The moderator has also provided a comment that is only visible to contributors
+                of the ${reviewable.provider.preprint_word}, and not to others.
+            % endif
+        </p>
+        <p>
+            You will ${'not receive ' if no_future_emails else 'be automatically subscribed to '}future
+            notification emails for this ${reviewable.provider.preprint_word}. Each
+            ${reviewable.provider.preprint_word} is associated with a project on the
+            Open Science Framework for managing the ${reviewable.provider.preprint_word}.
+            To change your email notification preferences, visit your
+            <a href="${'{}settings/notifications/'.format(domain)}">user settings</a>.
+        </p>
+        <p>
+            If you have been erroneously associated with "${reviewable.node.title}", then you
+            may visit the project's "Contributors" page and remove yourself as a contributor.
+        </p>
+        <p>
+        % if not is_rejected:
+            Now that you've shared your ${reviewable.provider.preprint_word}, take advantage of more OSF features:<br><br>
+                * Upload supplemental, materials, data, and code to the OSF project associated with your ${reviewable.provider.preprint_word}: ${reviewable.node.absolute_url}.
+                  Learn how: <a href="http://help.osf.io/m/preprints/l/685323-add-supplemental-files-to-a-preprint" target="_blank">help.osf.io</a><br><br>
+                * Preregister your next study and become eligible for a $1000 prize: <a href="https://goo.gl/Wj5oBB">osf.io/prereg</a><br><br>
+                * Or share on social media: Tell your friends through:
+                    <a href="${'https://twitter.com/home?status=Read%20my%20{word}%2C%20%E2%80%9C{title}%E2%80%9D%20on%20{name}%20{link}'.format(word=reviewable.provider.preprint_word,title=reviewable.node.title, name=reviewable.provider.name, link=reviewable.absolute_url)}" target="_blank">
+                        <img src="${settings.DOMAIN + 'static/img/fa-twitter-blue.png'}" alt="twitter" width="20" style="padding-left: 5px; vertical-align: bottom;">
+                    </a>
+                    <a href="${'https://www.facebook.com/sharer/sharer.php?u={link}%3Futm_source%3Dnotification%26utm_medium%3Demail%26utm_campaign%3Dpreprint_review_status'.format(link=reviewable.absolute_url)}" target="_blank">
```

Comment:
Other places we share from FB require a FB app id, are these used somewhere else like this or are they new? (if they work they work and that's fine, just wanted to confirm)

Reply:
Here, it's used differently. We pass facebook the preprint url with the campaign parameters.  Facebook goes and retrieves the preprint and since we have opengragh (og) tags in preprint detail metadata head tags, fcbk creates the post. Once posted clicking on the post takes the fcbk user to preprint detail page with the campaign parameters. That way ga knows to what campaign the visit counts. a similar use here though with goo.gl short urls: https://github.com/CenterForOpenScience/osf.io/blob/a0b5854b08353cd263b57604f816a52d0b1db0c9/website/templates/emails/prereg_challenge_accepted.html.mako#L17